hw/xilinx/ips/common/xlnx_axi_crossbar/config.tcl

```diff
@@ -18,9 +18,9 @@ set_property -dict [list CONFIG.PROTOCOL {AXI4} \
                          CONFIG.RUSER_WIDTH {0} \
                          CONFIG.BUSER_WIDTH {0} \
                          CONFIG.M00_A00_BASE_ADDR {0x0} \
-                         CONFIG.M01_A00_BASE_ADDR {0x100000} \
-                         CONFIG.M02_A00_BASE_ADDR {0x200000} \
+                         CONFIG.M01_A00_BASE_ADDR {0x10000} \
+                         CONFIG.M02_A00_BASE_ADDR {0x20000} \
                          CONFIG.M00_A00_ADDR_WIDTH {16} \
-                         CONFIG.M01_A00_ADDR_WIDTH {12} \
-                         CONFIG.M02_A00_ADDR_WIDTH {12} \
+                         CONFIG.M01_A00_ADDR_WIDTH {16} \
+                         CONFIG.M02_A00_ADDR_WIDTH {16} \
                          ] [get_ips $::env(IP_NAME)]
```

Comment:
Maybe we should add this file to the .gitignore, since it is automatically generated/configured by the config flow. Otherwise, if a user forget about running config, he would have a misaligned crossbar configuration without knowing it.

Reply:
Agree

Reply:
I agree but I would do this in a config-oriented PR to make it more general, e.g. the same goes for the .ld and device tree

Reply:
Po vrimm